types/index.d.ts

```diff
@@ -6,7 +6,7 @@
 // Copyright (c) Microsoft Open Technologies Inc
 // Licensed under the MIT license.
 // TypeScript Version: 2.3
-type channel = "loadstart" | "loadstop" | "loaderror" | "exit" | "message";
+type channel = "loadstart" | "loadstop" | "loaderror" | "exit" | "message" | string;
```

Comment:
this makes the type useless if we just use a `string`. Let's continue the discussion in #514 where I've added a question.